site_scons/buildutils.py

```diff
@@ -11,7 +11,7 @@
 from pathlib import Path
 import logging
 from typing import TYPE_CHECKING
-import collections
+from collections.abc import Mapping
```

Comment:
Now that you've confirmed the fix works, I think `from collections.abc import Mapping as MappingABC` would be more clear. Also need to confirm that this doesn't generate any warnings down to Python 3.6.